imu/icm20xxx.h

```diff
@@ -0,0 +1,45 @@
+/*
+ * Phoenix-RTOS
+ *
+ * IMU I2C driver - icm20xxx device driver
+ *
+ * Copyright 2022 Phoenix Systems
+ * Author: Mateusz Niewiadomski
+ *
+ * This file is part of Phoenix-RTOS.
+ *
+ * %LICENSE%
+ */
+#ifndef _PHOENIX_I2C_ICM20XXX_H
+#define _PHOENIX_I2C_ICM20XXX_H
+
+#include <stdint.h>
+#include <communication.h>
+
+/* size of all data accessible from icm20xxx */
+#define ICM20XXX_DATA_ALL_SIZE 14
+
+/* all supported members of icm20xxx family */
+enum icm_family { icm20948 };
```

Comment:
all of the below seems like internal stuff (apart from init/get all data) and should be moved to .c file (applies everywhere else also)

Reply:
Yes, thats true. Moved to .c file. Thanks!